src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center ;
+    background-repeat: no-repeat;
+
+    &--1,
+    &--2,
+    &--3,
+    &--4 {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
Consider adding stars styling from the previous task. The modifier should work properly

Reply:
Not fixed. Make sure that it will work correctly for stars--3, stars--2 and other modifiers

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3311/files#r1259830227
still not fixed, it should work with `stars--3, stars--2 and so on`, not only with 4